examples/12_css/src/components/App.tsx

```diff
@@ -1,9 +1,9 @@
 import classNames from 'classnames';
 import styles from './App.module.css';
 import './App.css';
-import { container } from './styles.css.js';
-import { Counter } from './Counter.js';
-import { Banner } from './Banner.js';
+import { container } from './styles.css';
```

Comment:
This causes a temporary error like this in DEV, when you reload the browser.

<img width="281" alt="Untitled" src="https://github.com/dai-shi/waku/assets/490574/f6b3f491-b3b7-4f96-bce1-36208b04c41e">

Adding back `.js` fixes it.

Do you have a better solution? @himself65